electron-core/src/electron/CapacitorElectronApp.ts

```diff
@@ -211,7 +211,7 @@ export class CapacitorElectronApp {
   /** Creates mainwindow and does all setup. _Called after app.on('ready') event fired._ */
   init() {
     const neededBrowserWindowConfig = {
-      show: false,
```

Comment:
This is set to false so the main content window is not shown while its content is loading. Showing a blank window is bad practice for desktop development.